_posts/2020-10-15-2.0.0-release.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Apache Arrow 2.0.0 Release"
+date: "2020-10-14 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+<!--
+
+To use this template:
+
+* Update all "XX" values with the appropriate numbers (you can get the resolved issues and contributors count from `_release/2.0.0.md`)
+* Fill in the various sections below. Note that the audience is the broader user community, not Arrow developers, so please write clearly using terms they will understand and care about. Delete any sections that don't have any content (as in, there are no changes to announce)
+* Delete this introductory comment
+
+ -->
+
+
+The Apache Arrow team is pleased to announce the 2.0.0 release. This covers
+over XX months of development work and includes [**XX resolved issues**][1]
+from [**XX distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+<!-- Acknowledge and link to any new committers and PMC members since the last release. See previous release announcements for examples. -->
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+For Arrow Flight, 2.0.0 mostly brings bugfixes. In Java, some memory leaks in `FlightStream` and `DoPut` have been addressed. In C++ and Python, a deadlock has been fixed in an edge case. Additionally, when supported by gRPC, TLS verification can be disabled.
+## C++ notes
+
```

Comment:
@pitrou @bkietz @emkornfield we need C++ notes. Presumably the nested parquet should be called out here, along with dataset writing, and any other highlights.

Reply:
yes, for parquet.  I think you can copy my comments above about parquet from the python section.